docs/packaging.rst

```diff
@@ -7,8 +7,8 @@ Downstream packagers often want to package Hypothesis. Here are some guidelines.
 The primary guideline is this: If you are not prepared to keep up with the Hypothesis release schedule,
 don't. You will annoy me and are doing your users a disservice.
 
-Hypothesis has quite a frequent release schedule. It's very rare that it goes a month without a release,
-and there are often multiple releases in a given month.
+Hypothesis has a very frequent release schedule. It's rare that it goes a week without a release,
+and there are often multiple releases in a given week.
 
 Many people not only fail to follow the release schedule but also seem included to package versions
```

Comment:
While you are here... "included" was probably supposed to be "inclined". But you might want to just get rid of the paragraph and one below altogether. I think some of the appearance of packaging stuff that is months out of date is really auto- activities, or global fixes for packaging problems by release engineering or what not.